airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresSourceTest.java

```diff
@@ -544,4 +544,44 @@ private JsonNode buildConfigEscapingNeeded() {
         JdbcUtils.SSL_KEY, "false"));
   }
 
+  @Test
+  public void tableWithNullValueCursorShouldThrowException() throws SQLException {
```

Comment:
Lets add a similar test where we have a VIEW with null value for cursor column

Reply:
Added